scripts/hitl_smoke_test_jenkins.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python2
```

Comment:
How is this script different from the non jenkins version?  Can we unify the scripts by adding a command line option?

Reply:
Actually, since the HITL trigger was moved into the groovy lib, we can delete this, right?

Reply:
Never mind... forgot that we're invoking this script, not triggering HITL directly.  I think this is something that's a good candidate for moving into our groovy lib, so that PFWP and PBR can share code.

In the mean time, I think we can just delete the Travis version of this script.

Reply:
Deleted the travis version of the script.

The script is actually called from the groovy hitl lib class; that's not too elegant as now the lib class requires this script to be present (i.e. it only works in a pbr workspace), but I didn't want to rewrite the scripts too much. I can open a separate ticket to further restructure it.

Reply:
👍